src/audio/multimediaaudioplayer.hh

```diff
@@ -27,7 +28,7 @@ private slots:
 
 
 private:
-  QBuffer audioBuffer;
+  QPointer< QBuffer > audioBuffer;
```

Comment:
Change this to std::unique_ptr/QScopedPointer instead?

QPointer doesn't manage lifetime but QScopedPointer does.

There is a small leak here.